### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Joey690152**
Updated colours for Nav as well as removing underline and adding a border bottom instead... hopefully to use a nice effect later on in the styling process

### CODE REVIEW

File: src/css/tailwind.css [hunk 1]
```diff
@@ -3,7 +3,8 @@
...
 
 a {
-    @apply font-bold text-green underline;
-    text-decoration-color: #60FCA2;
-    text-decoration-thickness: 3px;
-}
\ No newline at end of file
+  @apply font-bold text-green;
+  text-decoration-color: #3e4653;
+  text-decoration-thickness: 3px;
```
Comment (PatrickWhitehouse) [inline src/css/tailwind.css:8]: I'd remove 7 & 8 too.

**Review by PatrickWhitehouse — CHANGES_REQUESTED** [review]
Added a note to that file

File: src/css/tailwind.css (hunk 1 above)
Comment (Joey690152) [reply src/css/tailwind.css:8]: No problem. i will do tomorrow

Comment (Joey690152) [reply src/css/tailwind.css]: i believe this has now been amended correctly.

**Review by PatrickWhitehouse — APPROVED** [review]
Nice one.

**Pull request merged**